### PR TITLE
Cews count down

### PR DESCRIPTION
**Pull request by cews7**
added countdown functionality for project show page
made buttons functional 
removed comment form for guests

### CODE REVIEW

File: app/helpers/project_helper.rb [hunk 1]
```diff
@@ -1,5 +1,16 @@
...
+    num_days  = (end_date - now_date).to_i
+    hours     = date.hour
+    now_hours = Time.now.hour
+    num_hours = hours - now_hours
+    if num_hours < 0
+      num_days  = num_days - 1
+      num_hours = 24 + num_hours
+    end
+    time_remaining = " #{num_days} days, #{num_hours} hours remaining to fund!"
+    time_remaining
```
Comment (CjMoore) [inline app/helpers/project_helper.rb]: We should refactor this later.

File: app/views/projects/show.html.erb [hunk 2]
```diff
@@ -40,7 +40,7 @@
     <% if current_user %>
       <%= link_to "Fund", "#", class: "btn right btn-color" %>
     <% else %>
-      <%= link_to "Register/Sign in to fund", '#', class: "btn right btn-color" %>
+      <%= link_to "Register/Sign in to fund", login_path, class: "btn right btn-color" %>
```
Comment (CjMoore) [inline app/views/projects/show.html.erb]: 🔥

File: app/helpers/project_helper.rb (hunk 1 above)
Comment (scmountain) [inline app/helpers/project_helper.rb]: looks good, refactor in our "spare" time

File: db/seeds.rb [hunk 3]
```diff
@@ -18,7 +15,7 @@ def generate_projects
       name: Faker::Name.name,
       description: Faker::StarWars.wookie_sentence,
       total: Faker::Number.between(1, 100),
-      time: Faker::Time.between(DateTime.now - 1, DateTime.now),
+      time: Faker::Time.forward(Random.new.rand(3..30), :morning),
```
Comment (scmountain) [inline db/seeds.rb]: never seent his before... cool

**Pull request merged**